src/com/google/enterprise/adaptor/DocIdSender.java

```diff
@@ -356,7 +368,7 @@ public GroupPrincipal pushGroupDefinitions(
       try {
         log.info("sending groups to GSA host name: " + config.getGsaHostname());
         fileSender.sendGroups(feedSourceName,
-            groupsDefXml, config.isServerToUseCompression());
+            groupsDefXml, config.isServerToUseCompression(), incremental);
```

Comment:
This parameter order feels a little random, but so do the other orders. Moving incremental after feedSourceName feels slightly better to me (sort of the reverse of pushGroupDefinitions and its friends).